assets/js/editor-components/search-list-control/item.tsx

```diff
@@ -22,13 +22,22 @@ const Count = ( { label }: { label: string | React.ReactNode | number } ) => {
 	);
 };
 
+type SearchListItemPropsWithBreadcrumb = SearchListItemProps & {
+	breadcrumbs: string[];
+};
+
+const checkHasBreadCrumbs = (
```

Comment:
Typeguard allows is to properly narrow the type to call `getBreadcrumbsForDisplay`